app/assets/javascripts/custom_play_button.js

```diff
@@ -25,12 +25,12 @@ $(document).ready(function() {
 
   function playVideo(video) {
     pauseOtherVideos(video);
-    $(video).closest('.video-container').children('.play-button').css('visibility', 'hidden');
+    $(video).closest('.video-container').children('.play-button').css('opacity', '0');
```

Comment:
I have an 🐘 and a more serious concern:

🐘 : Because you're changing object properties in JavaScript, the preferred method is to use object notation, so it might be better to have it look like this:

```
$(video).closest('.video-container').children('.play-button').css({ opacity: 0 });
```

Now the real thing. By reducing the opacity, the player is still on the page. It's theoretically possible for a user to interact with this menu by clicking on it, even if it doesn't _seem_ to be there. It's probably better to use a set of properties like this and then change the class instead of assigning the CSS:

```
.play-button {
  &.hide-while-playing {
    display: none;
    pointer-events: none;
  }
}
```

Reply:
thats intended behaviour. This line only hides the play button, but we need it to remain there because when it gets clicked again the video should pause. The old method of visibility hidden worked fine in most browsers but safari had issues with it so the opacity change is a more browser cross compatible solution

Reply:
👍

Reply:
I like what you are saying however this implementation means the custom icon button does not get removed(hidden) at all which is the main requirement.